src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -292,7 +292,8 @@ public enum CaseEvent {
     SEND_CVP_JOIN_LINK(USER),
     SET_LIP_RESPONDENT_RESPONSE_DEADLINE(CAMUNDA),
     NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA),
-    NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA);
+    NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA),
+    SEND_HEARING_TO_LIP_DEFENDANT(CAMUNDA);
```

Comment:
Do we also need events for hearing to Claimant LIP + Final orders for claimant & defendant?

Reply:
I will confirm if it is included in these developments

Reply:
Included in code